setup.py

```diff
@@ -115,7 +115,8 @@ def run_tests(self):
         'pyyaml',
         'argparse',
         'argcomplete',
-        'future'
+        'future',
+        'pyBarcode==0.8b1'
```

Comment:
OK, then we can keep it like this.

After reading [pyBarcode#16](https://bitbucket.org/whitie/python-barcode/issues/16/pypi-08-release-request) it looks like this project is discontinued. However it seems to be continued in [this project](https://github.com/kxepal/viivakoodi).
So maybe we can switch the dependency and then improve this project.